### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by RoyTheReal** [Sketchware-Pro/Sketchware-Pro#978] [General]
When will the "Fix custom component" feat be merged with the Sketchware material redesign along with others

**Comment by ilyassesalama** [comment]
That pull request is still open and hasn't been merged into the main branch since it needs to be reviewed by the team. In the meantime, keep an eye on the main branch for changes because as soon as a new change is merged into main, I'll make sure to keep the redesign branch up-to-date.

**Reply by RoyTheReal** [reply]
Thank you so much, awesome to hear. I don't understand GitHub that much so thanks :)

**Reply by elfilibusterismo** [reply]
Have you tried the 'Fix custom component'?

**Reply by RoyTheReal** [reply]
Yes sir and it seems to work great. Good job

**Comment by ilyassesalama** [comment]
The pull request you mentioned has been merged into the redesign branch.

**Reply by elfilibusterismo** [reply]
He meant the custom component not custom variable.

**Reply by RoyTheReal** [reply]
You merged the custom component fix? Thanks

**Reply by RoyTheReal** [reply]
The custom component fix hasn't been merged yet sir.

**Comment by RoyTheReal** [comment]
The custom component fix still hasn't been merged

**Reply by j-px** [reply]
can you please be more patient and stop spamming?
